### PR TITLE
Use alembic directly, without flask-migrate

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

File: rest-service/manager_rest/test/base_test.py [hunk 1]
```diff
@@ -424,9 +425,10 @@ def _insert_default_permissions():
 
     @staticmethod
     def _handle_default_db_config():
-        Migrate(app=server.app, db=server.db)
+        cfg = current_app.extensions['migrate'].migrate.get_config(
+                MIGRATION_DIR)
         try:
-            upgrade(directory=MIGRATION_DIR)
+            alembic_cmd.upgrade(cfg, revision='head', sql=False, tag=None)
         except sqlalchemy.exc.OperationalError:
```
Comment (mateumann) [inline rest-service/manager_rest/test/base_test.py:432]: there's going to be a different error, which is not (yet) handled

**Comment by geokala** [comment]
This has been open for more than six months so I'm closing it. If it's still wanted, please reopen it and ping me so we can get it reviewed.

**Comment by geokala** [comment]
This has been open for more than six months so I'm closing it. If it's still wanted, please reopen it and ping me so we can get it reviewed.

**Comment by geokala** [comment]
This has been open for more than six months so I'm closing it. If it's still wanted, please reopen it and ping me so we can get it reviewed.

**Pull request closed without merging**